src/Config/Parser/MetadataParser/ClassesTypesMap.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Overblog\GraphQLBundle\Config\Parser\MetadataParser;
+
+class ClassesTypesMap
+{
+    protected array $classesMap = [];
```

Comment:
```suggestion
    /*
     * @var array{class: string, type: string}[]
     */
    protected array $classesMap = [];
```

If the array consists of only several keys and you know them in advance, then you can describe its exact shape ([documentation](https://phpstan.org/writing-php-code/phpdoc-types#array-shapes)). This makes the code more robust and gives more information for developers, but it's totally optional. You don't have to accept this suggestion.